apps/condo/pages/onboarding.tsx

```diff
@@ -55,15 +58,18 @@ const OnBoardingPage: IOnBoardingIndexPage = () => {
                                 {onBoardingSteps.length > 0 && !get(onBoarding, 'completed')
                                     ? (
                                         <Row gutter={[0, 0]}>
-                                            {onBoardingSteps.sort((leftStep, rightStep) => leftStep.order > rightStep.order ? 1 : -1)
+                                            {onBoardingSteps
```

Comment:
it's better to memoize such calculations.

Reply:
Fixed: sorting was memoized.